Makefile

```diff
@@ -1,12 +1,17 @@
-PACKAGES := $(shell glide novendor)
+PACKAGES := $(filter-out ., $(shell glide novendor))
```

Comment:
We should leave `.` in `PACKAGES` since it's also used for the test command and we want to run tests for the main package too.